developer/frameworks/articles/02-dependency-injection/02-declarative-services/99-converting-a-service-builder-module-from-spring-di-to-osgi-ds.markdown

```diff
@@ -36,18 +36,19 @@ straightforward.
 
 ## Step 2: Update your Spring bean classes
```

Comment:
@Preston-Crary - Should you run Service Builder before updating your Spring Bean classes?

Reply:
We've found it easier to make the manual changes in place first so you can reference the code before rebuilding, some of the code we reference in the guide is removed after rebuilding. Either way works it's just this way it's easier to copy-paste the manual changes over.  If creating a project for the first time, these values are added for you.